management/ia/templates_test.go

```diff
@@ -24,6 +25,85 @@ import (
 // Note: Even though the IA services check for alerting enabled or disabled before returning results
 // we don't enable or disable IA explicit in our tests since it is enabled by default through
 // ENABLE_ALERTING env var.
+func assertTemplate(t *testing.T, name, expr string, expectedTemplate alert.Template, listTemplates []*templates.TemplatesItems0) {
```

Comment:
Can we use `expectedTemplate.Name/Expr` instead of explicit arguments?

Reply:
yeah, no idea why i did it :facepalm: